src/lib/storage/abstract_device_factory.rb

```diff
@@ -163,8 +165,12 @@ def build_tree_recursive(parent, name, content)
         end
       end
 
+      # rubocop:disable Lint/UselessAccessModifier
+
       protected
 
+      # rubocop:enable Lint/UselessAccessModifier
+
```

Comment:
I agree with you. But if I remember properly, the convenience of using abstract_method vs these comments was already discussed. Let's not restart that discussion (at least here).